app/secrets.py

```diff
@@ -0,0 +1,24 @@
+from app.missing_secret_exception import MissingSecretException
+
+EXPECTED_SECRETS = [
+    "EQ_SUBMISSION_SDX_PUBLIC_KEY",
+    "EQ_SUBMISSION_SR_PRIVATE_SIGNING_KEY",
+    "EQ_SUBMISSION_SR_PRIVATE_SIGNING_KEY_PASSWORD",
+    "EQ_USER_AUTHENTICATION_RRM_PUBLIC_KEY",
+    "EQ_USER_AUTHENTICATION_SR_PRIVATE_KEY",
+    "EQ_USER_AUTHENTICATION_SR_PRIVATE_KEY_PASSWORD",
+    "EQ_USER_AUTHENTICATION_SR_PUBLIC_KEY",
+    "EQ_SERVER_SIDE_STORAGE_USER_ID_SALT",
+    "EQ_SERVER_SIDE_STORAGE_USER_IK_SALT",
+    "EQ_SERVER_SIDE_STORAGE_ENCRYPTION_USER_PEPPER",
+    "EQ_SECRET_KEY",
+    "EQ_RABBITMQ_URL",
+    "EQ_RABBITMQ_URL_SECONDARY",
+    "EQ_SERVER_SIDE_STORAGE_DATABASE_URL",
+]
+
+
+def validate_required_secrets(secrets):
+    for required_secret in EXPECTED_SECRETS:
+        if required_secret not in secrets:
+            raise MissingSecretException("Missing Secret [{}]".format(required_secret))
```

Comment:
I don't think we need to have a `MissingSecretException`, we could just use `Exception`. Also, the required_secret bit doesn't get printed, you just get `app.missing_secret_exception.MissingSecretException`. 

Replacing `MissingSecretException` with `Exception` works and prints something like `Exception: Missing Secret [EQ_SERVER_SIDE_STORAGE_DATABASE_URL]`